src/coreclr/src/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.CoreCLR.cs

```diff
@@ -305,6 +305,8 @@ internal unsafe struct MethodTable
         public ushort InterfaceCount;
         [FieldOffset(ParentMethodTableOffset)]
         public MethodTable* ParentMethodTable;
+        [FieldOffset(ElementTypeOffset)]
+        public nuint ElementType;
```

Comment:
`void*` would be more appropriate type for this. This is not a number.

Reply:
It is a handle, so I picked `nuint`.  `void*` will work too.

Reply:
We use `void*` for type handle in all other places.